modules/serving-transport-encryption-configuring.adoc

```diff
@@ -0,0 +1,78 @@
+:_content-type: PROCEDURE
+[id="serving-transport-encryption-configuring_{context}"]
+= Configuring transport encryption
+
+Configuring transport encryption consists of two parts:
+
+. Specifying ClusterIssuer to use:
+
+`clusterLocalIssuerRef`: issuer for cluster-local-domain certificates used for ingress.
+
+`systemInternalIssuerRef`: issuer for certificates for system-internal-tls certificates used by Knative internal components.
+
+. Specifying transport encryption features to use:
+
+`cluster-local-domain-tls`: Enables the transport encryption feature for cluster-local domains
+
+`system-internal-tls`: Enables the transport encryption feature for OpenShift Serverless Serving internal components.
+
+.Procedure
+
+. Enable `transport-encryption` in `KnativeServing`:
+
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  ...
+  config:
+    certmanager:
+      clusterLocalIssuerRef: |
+        kind: ClusterIssuer
+        name: knative-serving-ca-issuer <1>
+      systemInternalIssuerRef: |
+        kind: ClusterIssuer
+        name: knative-serving-ca-issuer <2>
+    network:
+      cluster-local-domain-tls: Enabled <3>
+      system-internal-tls: Enabled <4>
+----
+<1> Define the ClusterIssuer for each feature. The same or individual ClusterIssuers can be used.
+<2> Define the ClusterIssuer.
+<3> Enabling the `cluster-local-domain-tls` feature. This and other features can be enabled or disabled individually.
+<4> Enabling the `system-internal-tls` feature.
+
+. Apply the `KnativeServing` resource:
+
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
```

Comment:
Ideally, we'd describe how to change the default Ingress certificate here as described in https://github.com/openshift/openshift-docs/pull/79108#discussion_r1786116937

Reply:
Done in https://github.com/openshift/openshift-docs/pull/82435/commits/361a5c167989bae5689c59c9dd3881d7007352aa

Reply:
This part looks good to me!

Reply:
@mgencur Thanks! What about the section as a whole?